src/components/molecules/Alert/Alert.tsx

```diff
@@ -56,31 +40,63 @@ export const Alert = ({
         }
     };
 
-    return isVisible ? (
+    const alertIcon = alertIconMapping[severity];
+
+    if (!isVisible) return null;
+
+    return (
         <section
             aria-label={severity}
             role='alert'
-            className={clsx('rounded-xl', {
-                'border-2 border-yellow bg-yellow/20': variant === 'outlined', // should be changed to handle different colors
-                'bg-[rgba(41, 45, 63, 0.60)] border border-white-10 shadow-tab':
-                    variant === 'solid',
-            })}
+            className={clsx(
+                'flex w-full flex-row items-start justify-between gap-1 rounded-md border px-2.5 text-2xs text-neutral-50 laptop:text-xs',
+                severityStyle[severity],
+                {
+                    'py-1.5': !subtitle,
+                    'py-2': subtitle,
+                }
+            )}
         >
-            <div
-                className={clsx(
-                    'flex w-full flex-row items-center justify-between gap-1 rounded-xl px-5 py-3',
-                    {
-                        'bg-gradient-to-b from-[rgba(111,116,176,0.35)] to-[rgba(57,77,174,0)]':
-                            variant === 'solid',
-                    }
+            <div className='flex items-start gap-1.5 pr-4 laptop:gap-2'>
```

Comment:
right padding is it necessary?

Reply:
there is not much clarity in the designs regarding the horizontal gap between the close button and rest of the elements

Reply:
In the case of very long titles and subtitles, I chose to put right padding so subtitles break to next line earlier, so content is not too close to the close button

Reply:
Yeah there isn't much clarity there. I left it to practicality with that right padding. Let me know what you think